src/poetry/packages/locker.py

```diff
@@ -40,8 +41,8 @@
 logger = logging.getLogger(__name__)
 _GENERATED_IDENTIFIER = "@" + "generated"
 GENERATED_COMMENT = (
-    f"This file is automatically {_GENERATED_IDENTIFIER} by Poetry and should not be"
-    " changed by hand."
+    f"This file is automatically {_GENERATED_IDENTIFIER} by Poetry (version"
+    f" {__version__}) and should not be changed by hand."
```

Comment:
```suggestion
    f"This file is automatically {_GENERATED_IDENTIFIER} by Poetry"
    f" {__version__} and should not be changed by hand."
```

Should be sufficient and is consistent with what we write into the INSTALLER file in #6205.